homeassistant/components/sense/sensor.py

```diff
@@ -41,25 +50,13 @@
     TO_GRID_NAME,
 )
 
-
-class SensorConfig:
-    """Data structure holding sensor configuration."""
-
-    def __init__(self, name, sensor_type):
-        """Sensor name and type to pass to API."""
-        self.name = name
-        self.sensor_type = sensor_type
-
-
-# Sensor types/ranges
-ACTIVE_SENSOR_TYPE = SensorConfig(ACTIVE_NAME, ACTIVE_TYPE)
-
 # Sensor types/ranges
 TRENDS_SENSOR_TYPES = {
-    "daily": SensorConfig("Daily", "DAY"),
-    "weekly": SensorConfig("Weekly", "WEEK"),
-    "monthly": SensorConfig("Monthly", "MONTH"),
-    "yearly": SensorConfig("Yearly", "YEAR"),
+    Scale.DAY: "Daily",
+    Scale.WEEK: "Weekly",
+    Scale.MONTH: "Monthly",
+    Scale.YEAR: "Yearly",
+    Scale.CYCLE: "Bill",
```

Comment:
Why is this one added now

Reply:
The "Cycle" scale was one that was available before, just overlooked.  With the conversion to the new Scale enum used by the new library, I just added all the scales